grails-web-boot/src/test/groovy/grails/boot/GrailsSpringApplicationSpec.groovy

```diff
@@ -34,8 +32,7 @@ class GrailsSpringApplicationSpec extends Specification{
     }
 
 
-    @Configuration
-    @EnableWebMvc
+    @SpringBootApplication
```

Comment:
Any idea why these tests didn't do this before?  I assume it was to limit the amount initialized?  

If you add @EnableAutoConfiguration would this fix them too?

Reply:
yes, but why add @EnableAutoConfiguration when you can just add @SpringBootApplication which automatically applies @Configuration and @ EnableAutoConfiguration